### PR TITLE
Bug 567032 issue floating license configuration wizard

### PR DESCRIPTION
**Pull request by eparovyshnaya**
Finish the wizard. The license is still to be issued.

Signed-off-by: eparovyshnaya <elena.parovyshnaya@gmail.com>

### CODE REVIEW

File: bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/loc/dashboard/ui/wizards/floating/IssueFloatingLicenseWizard.java [hunk 1]
```diff
@@ -12,14 +12,24 @@
...
+import org.eclipse.passage.lic.internal.jface.dialogs.licensing.DiagnosticDialog;
+import org.eclipse.passage.loc.dashboard.ui.wizards.license.WizardInfoBar;
+import org.eclipse.passage.loc.internal.api.IssuedFloatingLicense;
 import org.eclipse.passage.loc.internal.dashboard.ui.i18n.IssueLicensePageMessages;
 
 public final class IssueFloatingLicenseWizard extends Wizard {
 
 	private final IEclipseContext context;
 	private final FloatingDataPack initial;
+	private Supplier<FloatingLicensePack> license;
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/loc/dashboard/ui/wizards/floating/IssueFloatingLicenseWizard.java:31]: final

File: bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/loc/dashboard/ui/wizards/floating/IssueFloatingLicenseWizard.java [hunk 2]
```diff
@@ -12,14 +12,24 @@
...
+	private Supplier<ServerConfigsRequest> personals;
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/loc/dashboard/ui/wizards/floating/IssueFloatingLicenseWizard.java:32]: final

File: bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/loc/dashboard/ui/wizards/floating/IssueLicensePackPage.java [hunk 3]
```diff
@@ -94,12 +84,22 @@ public void createControl(Composite parent) {
 		Dialog.applyDialogFont(composite);
 	}
 
-	@Override
-	public void setVisible(boolean visible) {
-		if (visible) {
-			init();
+	private void init() {
+		boolean render = license == null;
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/loc/dashboard/ui/wizards/floating/IssueLicensePackPage.java:88]: how about to inline this variable?

File: bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/loc/dashboard/ui/wizards/floating/IssueFloatingLicenseWizard.java (hunk 1 above)
Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/loc/dashboard/ui/wizards/floating/IssueFloatingLicenseWizard.java:31]: Unfortunately, no

File: bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/loc/dashboard/ui/wizards/floating/IssueLicensePackPage.java (hunk 3 above)
Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/loc/dashboard/ui/wizards/floating/IssueLicensePackPage.java:88]: There is a temporal coupling here: after the second line of the function executes, the `license` is never null. We track `first render` this way - due to EMF Forms rendering specifics.

File: bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/loc/dashboard/ui/wizards/floating/IssueFloatingLicenseWizard.java (hunk 2 above)
Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/loc/dashboard/ui/wizards/floating/IssueFloatingLicenseWizard.java:32]: They cannot be final.

**Review by ruspl-afed — APPROVED** [review]

**Pull request merged**